src/clj/ataru/forms/form_payment_info.clj

```diff
@@ -72,25 +72,35 @@
     (throw (user-feedback-exception
              (str "Maksutiedot virheelliset: " [payment-type processing-fee decision-fee])))))
 
+(defn- add-vat
+  "Adds vat to form if valid."
+  [form vat]
+  (if (and (some? vat) (< vat 0.00M))
+    (user-feedback-exception
+      (str "ALV virheellinen: " [vat]))
+    (assoc-in form [:properties :payment :vat] (str vat))))
```

Comment:
Tässä lisätään formiin tuo vat aina, vaikka sen arvo olisi nil (jolloin arvoksi tulee tyhjä stringi) - pitäisikö näin, vai vielä toinen iffittely jolla palautetaan formi muuttumattomana jos alvia ei ole annettu parametrina?